packages/forms/backend/sirius-components-collaborative-forms/src/main/resources/schema/form.graphqls

```diff
@@ -367,7 +367,10 @@ type TreeNode {
   label: String!
   kind: String!
   iconURL: [String!]!
+  iconEndURL: [[String!]!]!
```

Comment:
It seems you've forgotten to also update the name of the field here to `endIconsURL`. Sorry I should have mentioned it in the last review, that the rename should have been done everywhere.